source/extensions/filters/http/grpc_web/grpc_web_filter.cc

```diff
@@ -17,6 +19,45 @@ namespace Extensions {
 namespace HttpFilters {
 namespace GrpcWeb {
 
+namespace {
+
+// Bit mask denotes a trailers frame of gRPC-Web.
+constexpr const uint8_t GRPC_WEB_TRAILER = 0b10000000;
+constexpr const uint64_t MAX_GRPC_MESSAGE_LENGTH = 1024;
+
+const absl::flat_hash_set<std::string>& gRpcWebContentTypes() {
```

Comment:
Should this be a unit tested utility?

Also is there a reason for the move?

Reply:
No specific reason, but it is tested indirectly, but yeah good point on adding unit tests. Thanks!

Reply:
if we can minimize diffs to that'd be helpful!